twifbeddit-app/backend/src/users.js

```diff
@@ -29,3 +29,60 @@ exports.POST = async (_, event) => {
 
     };
 };
+
+// TODO: Test this. Gets the topics that the user is following.
+exports.get_topics_that_the_user_is_following = async (_, event) => {
```

Comment:
This isn't necessary. A GET on /user will give the whole user object which will have this.